.github/workflows/ci.yaml

```diff
@@ -19,6 +19,7 @@ on:
       - synchronize
       - labeled
       - unlabeled
+      - closed
```

Comment:
we should split these changes into a workflow file.

Reply:
Like this:

https://github.com/cryostatio/cryostat/blob/main/.github/workflows/image-cleanup.yml